### ISSUE TITLE
vitepress support

### ISSUE THREAD

**Issue by SamWangCoder** [vue-styleguidist/vuepress-plugin-live#58]
firstly，thank you for your work.
Is there a plan to support vitepress？

**Comment by elevatebart** [comment]
There is a plan, but it will not be called `vuepress-plugin-live`

**Comment by elevatebart** [comment]
There is one more thing I have not figured out how to do.

To compile templates, Vue needs its bundle time version in the browser. To do that, we set an alias of `vue` to `vue/dist/vue.esm-bundler.js`. 
Sadly, import alias do not seem to be implemented in VitePress (yet). I believe we would need some custom plugin.
Remeber. that VitePress is still alpha 😉

**Comment by SamWangCoder** [comment]
Yes! I have tried to merge your [vite-vue-live](https://github.com/elevatebart/vite-vue-live) to `vitepress` proj but failed. Vitepress ignore the config of `vite.alias`. 
So I try to merge [Ant Design Vue Playground](https://github.com/vueComponent/sfc-playground) into my proj, it works. But can not import local components into repl real-time. 
In consideration of `VitePress` still a alpha version, I'll try create a plugin later.
